src/TimeWarpEventSet.cpp

```diff
@@ -120,7 +136,10 @@ std::shared_ptr<Event> TimeWarpEventSet::getEvent (unsigned int thread_id) {
 
     unsigned int scheduler_id = worker_thread_scheduler_map_[thread_id];
```

Comment:
unsigned int scheduler_id = worker_thread_scheduler_map_[thread_id]
You'll see this code used at several places in the TimeWarpEventSet.cpp. But since it is a one-to-one mapping, scheduler_id equals thread_id. It is not absolutely necessary but it might provide a minor performance boost if you deactivate this call using the compiler flag. The underlying data structure for worker_thread_scheduler_map_ has O(1) lookup complexity - so it is not a major lookup penalty.

Reply:
I modified it. Wilsey would like any optimization possible.